packages/apollo-voyager-keycloak/package.json

```diff
@@ -23,7 +23,7 @@
     "test": "echo \"Error: run tests from root\" && exit 1"
   },
   "dependencies": {
-    "@aerogear/apollo-voyager-server": "1.0.0",
+    "@aerogear/apollo-voyager-server": "file:../apollo-voyager-server",
```

Comment:
This will cause havoc on release time, however this could be swapped by lerna for release time. What is the context/problem for this change. Just for my information

Reply:
Seeing it recommended in many places. It basically reduces the dependency on using the `lerna bootstrap` command.

https://github.com/lerna/lerna/issues/1449#issuecomment-414713331
https://github.com/lerna/lerna/issues/1462#issuecomment-397154543

I am going to test the old way again and see can I get it working.